pages/agent/v3/securing.md

```diff
@@ -40,7 +40,7 @@ Command line evaluation can be disabled by setting [`no-command-eval`](/docs/age
 
 > 🚧 Custom hooks and environment variables
 > If you have a custom `command` hook, using `no-command-eval` will have no effect on your command execution. See [Allowing a list of plugins](#allowing-a-list-of-plugins) and [Custom bootstrap scripts](#customizing-the-bootstrap) for examples of how to completely lock down your agent from arbitrary code execution.
->
+><p>
```

Comment:
Thanks @rianmcguire , this makes the callout look a little clearer.

I'd only recommend adding a single space between the initial `>` and `<p>` to make it clearer that these are separate pieces of markup.

```suggestion
> <p>
```